401-react-clickable-grid/src/js/GridInput.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * 单元格行数输入组件
+ */
+
+var GridInput = React.createClass({
+    componentDidMount: function () {
+        // input组件挂载时选中框内文字，便于修改
+        this.refs.rowNumInput.select();
```

Comment:
is there way to use autofocus, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input ?